lib/p2p/Pool.ts

```diff
@@ -796,6 +801,7 @@ class Pool extends EventEmitter {
         connectedNodesInfo.push({
           nodePubKey: connectedPeer.nodePubKey!,
           addresses: connectedPeer.addresses,
+          inbound: connectedPeer.inbound,
```

Comment:
I believe this is redundant - no need to send a `NodesPacket` containing the `inbound` field.